gloss/Graphics/Gloss/Internals/Interface/Backend/Types.hs

```diff
@@ -64,6 +64,9 @@ class Backend a where
         -- | Function that returns (width,height) of the window in pixels.
         getWindowDimensions        :: IORef a -> IO (Int,Int)
 
+        -- | Function that returns (width,height) of a fullscreen window in pixels.
+        getScreenSize              :: IORef a -> IO (Int,Int)
```

Comment:
Now this is part of the backend type class.

Previously, with GLFW, we were asking GLUT for the screen size, but then, when GLFW created the window, GLFW set its own screen size. So different backends were doing different parts of the job.

I think that, conceptually, it's a back-end operation anyway.